golos.charge/golos.charge.cpp

```diff
@@ -0,0 +1,122 @@
+#include "golos.charge.hpp"
+#include <eosio.token/eosio.token.hpp>
+#include <golos.vesting/golos.vesting.hpp>
+
+namespace golos {
+using namespace eosio;
+using namespace atmsp::storable;
+
+extern "C" {
+    void apply(uint64_t receiver, uint64_t code, uint64_t action) {
```

Comment:
btw, why use own hand-written dispatcher instead of `EOSIO_DISPATCH` macro? just interesting

Reply:
for no reason, actually. 
fixed, thx.